api/config/config_test.json

```diff
@@ -5,7 +5,8 @@
     "billingAccountId": "014D91-FCB792-33D2C0",
     "billingProjectPrefix": "aou-test-f1-",
     "registeredDomainName": "all-of-us-registered-test",
-    "enforceRegistered": false
+    "enforceRegistered": false,
+    "jupyterUserScriptUri": "gs:\/\/all-of-us-workbench-test-scripts/setup_notebook_cluster.sh"
```

Comment:
Nit (and this goes for enforceRegistered too, my bad): We should probably alphabetize. Goes for above too.

Reply:
Done